### ISSUE TITLE
Finalize Cleanup of performance timing tool

### ISSUE THREAD

**Issue by JTonda** [zowe/perf-timing#3]
-testing
-doc commands/usage doc
-merge to master
-implement gene's versioning (must be done in line with zowe/zowe-cli#135 so probably needs to be a separate issue)

Child of zowe/zowe-cli#133
Estimate 8

Depends on #2 

### To Do's:

- [x] Collect system information in results of each performance result
   Status: Complete
- [x] Consistency with measurements, function timers should store each call and relevant parameters.
   Status: Complete
- [x] Unit test application code. (Integration tests should be separate effort)
   Status: Complete
- [x] Document code in JS Doc
   Status: Complete
- [x] Create Usage Documents and Readme
   Status: Complete
- [x] Initial Publish to line with version scheme in zowe/zowe-cli#135, even if manual for now.
   Status: Complete
  - This can be replaced with the pipeline created out of the effort.
